drivers/amazonec2/amazonec2.go

```diff
@@ -903,15 +889,13 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) []*
 	hasDockerPort := false
 	hasSwarmPort := false
 	for _, p := range group.IpPermissions {
-		if p.FromPort != nil {
```

Comment:
This `p.FromPort != nil` line should not be removed.  It was introduced in `f17cce330c5b4b7a0b04d3c931d3a96cdff53476` to prevent a panic from occuring.